### PR TITLE
Add script to send installation analytics via MixPanel

### PR DESCRIPTION
**Pull request by textbook**
Thanks for contributing to postfacto. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:

    Introduces a new script, `mixpanel.sh`, that is used during deployment/upgrade to anonymously inform the maintainers that someone's using Postfacto.

* An explanation of the use cases your change solves

    Currently we don't know how many people are installing/upgrading Postfacto instances using these scripts, so it's hard to make sensible product decisions.

* Links to any other associated PRs

    Split out of #174 

* [x] I have reviewed the [contributing guide](https://github.com/pivotal/postfacto/blob/master/CONTRIBUTING.md)

* [x] I have made this pull request to the `master` branch

* [x] I have run all the tests using `./test.sh`.

* [x] I have added the [copyright headers](https://github.com/pivotal/postfacto/blob/master/license-header.txt) to each new file added

* [x] I have given myself credit in the [humans.txt](https://github.com/pivotal/postfacto/blob/master/humans.txt) file (assuming I want to)

### CODE REVIEW

**Comment by textbook** [comment]
Since the implementation in the other PR I switched to using an environment variable, `ANALYTICS`, rather than a flag `--{skip,send}-analytics` because I realised it would interfere with the other optional arguments (i.e. the CF domain).

**Comment by crswty** [comment]
A nice future improvement might be to have the ANALYTICS environment variable also set enable_analytics in config.js when it is set as on. That way users can set it in one place to enable for  both the install and the running app

File: deployment/mixpanel.sh [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+# You should have received a copy of the GNU Affero General Public License
+#
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+set -euo pipefail
+
+if [ $# -lt 1 ]; then
+  echo 'usage: ./mixpanel.sh <script-name> ...'
+  echo 'This will report the script execution to MixPanel.'
+  echo 'Use ANALYTICS=0 (skip) or ANALYTICS=1 (send) to avoid the manual check.'
```
Comment (crswty) [inline deployment/mixpanel.sh]: Consider renaming the variable to ENABLE_ANALYTICS and the value to true/false to be in line with the enable_analytics in config.js (even if they don't affect each other right now)

File: deployment/mixpanel.sh [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    echo 'about installs so that we can make Postfacto better for you and other users.'
+    echo ''
+    echo "To let us know you've installed Postfacto, we would like to send the following "
+    echo 'information via MixPanel:'
+    echo ''
+    echo "$PAYLOAD"
+    echo ''
+    echo -n 'Do you want to send this anonymous installation notification? [yN] '
+    read -r ACCEPT
+    if [[ $ACCEPT =~ 'y' ]]; then
```
Comment (crswty) [inline deployment/mixpanel.sh]: Any reason for regex here? Should this not be an exact check?

Comment (crswty) [reply deployment/mixpanel.sh]: i.e 'yno' should probably not send analytics

**Comment by crswty** [comment]
The only thing I think should change is the regex test. Other than that, all looks good (tested a little locally but didn't test it post package.sh)

**Pull request merged**